packages/slice-machine/src/components/SideNav/SideNavEnvironmentSelector.css.ts

```diff
@@ -67,3 +67,12 @@ export const actionRequiredLabel = style([
     fontWeight: 500,
   },
 ]);
+
+export const logInIcon = style([
+  sprinkles({
+    position: "relative",
+    width: 24,
+    height: 24,
+  }),
+  { left: "-1px" },
```

Comment:
The icon was visually off-centered by 1px when put into a square button.

It would always be off-centered regardless where it is used, so I refactored the icon's SVG to always be shifted 1px to the left and removed the `loginIcon` class.

With the 1px offset:
<img width="345" alt="image" src="https://github.com/prismicio/slice-machine/assets/8601064/f5e0c257-b4b5-41ea-aa03-647f2065aeb0">

Without the 1px offset:
<img width="321" alt="image" src="https://github.com/prismicio/slice-machine/assets/8601064/3bf94e06-a234-4a83-92a2-879379d85f0f">